core/src/main/java/io/parsingdata/metal/data/selection/ByType.java

```diff
@@ -16,26 +16,27 @@
 
 package io.parsingdata.metal.data.selection;
 
+import static io.parsingdata.metal.Util.checkNotNull;
+
 import io.parsingdata.metal.data.ParseGraph;
 import io.parsingdata.metal.data.ParseGraphList;
 import io.parsingdata.metal.data.ParseItem;
-
-import static io.parsingdata.metal.Util.checkNotNull;
+import io.parsingdata.metal.data.ParseItemList;
 
 public final class ByType {
 
     private ByType() {}
 
-    public static ParseGraphList getRefs(final ParseGraph graph) {
+    public static ParseItemList getRefs(final ParseGraph graph) {
```

Comment:
This is tangled with the main issue in several ways. In fact, the test code that uses it has already been removed since it were dodgy tests that wanted to work around *really* inspecting the data structure (instead doing some heuristic checks). I rewrote the test code to do a proper `ParseGraph` inspection. But I suppose `ParseGraphList` may as well go now...